Presentation.Web/Controllers/API/V1/GridLocalItContractRolesController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Core.DomainModel.ItContract;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using Core.Abstractions.Types;
+using Core.DomainModel.LocalOptions;
+using Core.DomainModel.Organization;
+using Core.DomainServices;
+using Core.DomainServices.Extensions;
+using Core.DomainServices.Generic;
+using Presentation.Web.Infrastructure.Attributes;
+
+namespace Presentation.Web.Controllers.API.V1
```

Comment:
I'm not sure on this one - should we not discourage the introduction of "V1" functionality? Could we not just introduce this one on V2 right away?